WordPress/Classes/Models/PostStatus.h

```diff
@@ -0,0 +1,13 @@
+#import <CoreData/CoreData.h>
+#import "Blog.h"
+
+@interface PostStatus : NSManagedObject
+
+@property (nonatomic, retain) NSString * label;
+@property (nonatomic, retain) NSString * name;
+@property (nonatomic, retain) NSNumber * isProtected;
+@property (nonatomic, retain) NSNumber * isPrivate;
+@property (nonatomic, retain) NSNumber * isPublic;
+@property (nonatomic, retain) Blog *blog;
```

Comment:
The PostStatus class was generated via CoreData, I believe CoreData handles this relationship itself transparently, I may be wrong. Also, PostCategory would have the same issue as PostStatus was modeled the same and both maintain strong inverse references to the blog.